Utilities/DavixAdaptor/interface/DavixFile.h

```diff
@@ -0,0 +1,50 @@
+#ifndef DAVIX_ADAPTOR_DAVIX_FILE_H
+#define DAVIX_ADAPTOR_DAVIX_FILE_H
+
+#include "Utilities/StorageFactory/interface/IOFlags.h"
+#include "Utilities/StorageFactory/interface/Storage.h"
+#include <davix.hpp>
+
+class DavixFile : public Storage {
```

Comment:
Please add `override`  to each of the virtual functions you are implementing for the base class `Storage`. This will help guarantee there is no typos.